### ISSUE TITLE
PROD_PASSWORD

### ISSUE THREAD

**Issue by upbound-web** [engageintellect/sveltekit-supabase#1]
Hey sorry I am new. I cant seem to find anything called prod password in supabase. I have got DB password. Just checking to see what it would be called in the supabase interface

**Comment by engageintellect** [comment]
If you click your db in supabase, then go to project settings > database, then scroll down a little, you will see an area to reset the password to the db:

<img width="1206" alt="Screenshot 2024-08-26 at 7 46 47 PM" src="https://github.com/user-attachments/assets/d8a34ffc-343d-4178-aabe-363fbb28ca24">

**Comment by upbound-web** [comment]
So that is the password it is asking for in env under PROD_PASSWORD?

**Comment by engageintellect** [comment]
> So that is the password it is asking for in env under PROD_PASSWORD?

Correct. And if I remember correctly - you should only need to use that env variable for dev.

for prod, I just have these (on vercel):
<img width="931" alt="Screenshot 2024-08-26 at 10 31 25 PM" src="https://github.com/user-attachments/assets/4162ced9-7f21-4448-b0a3-459be72cf681">
